canonical-web-and-design/maas-ui
# feat(ui): display notifications on the machine details network tab

## Done

- Migrate the notifications for the machine details network tab.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the network tab for a few machines. You should see notifications (sampledata has different variations).

Fixes: canonical-web-and-design/MAAS-squad#2154.
